src/ui.form-calendar/index.vue

```diff
@@ -202,10 +201,11 @@ defineOptions({ name: "UCalendar" });
 
 const props = defineProps({
   /**
-   * Calendar value (timestamp).
+   * Calendar value (Date, String).
```

Comment:
Calendar value in JS Date Objects or String formatted in provided `props.dateFormat`.